tkg/managementcomponents/helper.go

```diff
@@ -24,6 +25,13 @@ const (
 	packagePollTimeout  = 10 * time.Minute
 )
 
+func correctAddonsManagerPackageVersion(packageVersion string) string {
+	if !strings.HasSuffix(packageVersion, constants.DefaultVersionSuffix) {
```

Comment:
Can we only check for vmware instead of vmware.1. That way if vmware.x is there then we dont overwrite.

Reply:
wil use  regex to check for vmware.[\d] i.e matches... vmware.0 vmwware.1... vmware.100 if ever ) 

``` 
match, _ := regexp.MatchString("\\+vmware.\\d+$", packageVersion)